cfgov/v1/views/reports.py

```diff
@@ -106,3 +112,44 @@ def get_filename(self):
 
     def get_queryset(self):
         return Document.objects.all().order_by("-id").prefetch_related("tags")
+
+
+class ImagesReportView(ReportView):
+    header_icon = "image"
+    title = "All images"
+
+    list_export = [
+        "title",
+        "file",
+        "file_size",
+        "collection",
+        "tags.names",
+        "created_at",
+        "uploaded_by_user",
+    ]
+    export_headings = {
+        "title": "Title",
+        "file": "File",
+        "file_size": "Size",
+        "collection": "Collection",
+        "tags.names": "Tags",
+        "created_at": "Uploaded on",
+        "uploaded_by_user": "Uploaded by",
+    }
+
+    custom_field_preprocess = {
+        "tags.names": {"csv": process_tags},
+        "file": {"csv": construct_files_url},
+    }
+
+    template_name = "v1/images_report.html"
+
+    def get_filename(self):
+        """Get a better filename than the default 'spreadsheet-export'."""
+        return f"all-cfgov-images-{date.today()}"
```

Comment:
This might be outside the scope of this PR, but I wonder if we can reduce some duplication in all the `get_filename`s. Maybe instead of a custom name for each report we can just use the report's title in the filename, like `{self.title}-{date.today()}` (we'd probably want to lowercase and hyphenate the title in the filename, though). We could define that as a function up above and use it in all reports.

Reply:
I don't think this is outside the scope -- I can try to implement this within this PR.